listenbrainz/webserver/views/follow_api.py

```diff
@@ -0,0 +1,55 @@
+import ujson
+import listenbrainz.db.follow_list as db_follow_list
+import listenbrainz.db.user as db_user
+
+from flask import Blueprint, request, jsonify
+from listenbrainz.webserver.views.api import _validate_auth_header
+from listenbrainz.webserver.views.api_tools import log_raise_400
+from werkzeug.exceptions import NotFound, Forbidden, Unauthorized
+from listenbrainz.db.exceptions import DatabaseException
+
+follow_api_bp = Blueprint('follow_api_v1', __name__)
+
+@follow_api_bp.route("/save", methods=["POST", "OPTIONS"])
+def save_list():
+    creator = _validate_auth_header()
+    raw_data = request.get_data()
+    try:
+        data = ujson.loads(raw_data.decode("utf-8"))
+    except ValueError as e:
+        log_raise_400("Cannot parse JSON document: %s" % str(e), raw_data)
+
+    list_name = data['name']
```

Comment:
If name is missing we'll throw a 500. Should be 400. Let's improve the param checking, please.